specification/cognitiveservices/data-plane/Language/preview/2021-11-01-preview/analyzeconversations.json

```diff
@@ -495,19 +493,683 @@
         },
         "confidenceScore": {
           "format": "float",
-          "x-ms-client-name": "confidence",
           "description": "The entity confidence score.",
           "type": "number"
         },
-        "listKeys": {
+        "resolutions": {
           "type": "array",
-          "description": "List of keys.",
+          "items": {
+            "$ref": "#/definitions/BaseResolution"
+          }
+        },
+        "extraInformation": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/BaseExtraInformation"
+          }
+        }
+      }
+    },
+    "BaseExtraInformation": {
+      "type":"object",
+      "discriminator": "extraInformationKind",
+      "properties": {
+        "extraInformationKind":{
+          "type": "string",
+          "enum": [
+            "EntitySubtype",
+            "ListKeys"
+          ],
+          "x-ms-enum": {
+            "modelAsString": false
+          }
+        }
+      },
+      "required": [
+        "extraInformationKind"
+      ]
+    },
+    "EntitySubtype": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "x-ms-discriminator-value": "EntitySubtype",
+      "properties": {
+        "value": {
+          "type":"string",
+          "description": "The Subtype of an extracted entity type."
+        }
+      }
+    },
+    "ListKeys":{
```

Comment:
Same question here: should this change to "Resolutions" or something? Having consistency helps developer productivity.

Reply:
We have `resolutions` and `extraInformation` as two new property in our entity response object. `ListKeys` as explained in another comment is stored in the `extraInformation` array.